### ISSUE TITLE
Lack of Package Descriptions and their dependencies

### ISSUE THREAD

**Issue by jason-eu** [nikgapps/nikgapps.github.io#8]
On the page ;
https://nikgapps.com/downloads

The packages and what they contain (mostly) are listed.

However, in the example of the "Full" package, it is not listed on that page that it contains the like of YouTube and YouTube Music.
It lacks these details, so users end up installing apps that they don't want, and these apps are installed as System Apps.

While there exists, indeed, the options of using the NikGApps config and debloater files, you are still limited by the knowledge of what depends on what.

For example, users in the custom ROM community gravitate to alternatives because of this. In addition to just installing core or basic with a Setup Wizard add-on and realising later that they are missing other packages that they would need for the functionality that they would have if installing a higher level package.

It would benefit the project if the details of each package are outlined, whether clickable dropdowns or some other way as well as their relevant dependencies.

**Comment by nikhilmenghani** [comment]
@jason-stream thank you for your inputs, I will work on updating the website to reflect the package information right on the download page (easiest way is to add a hover message). 

other than that, Full package doesn't contain YouTube and YouTube Music in it, where did you find that Full Package contains it?

**Comment by jason-eu** [comment]
Thank you 🙏

NikGApps full from the sourceforge, here ;
https://sourceforge.net/projects/nikgapps/files/Releases/NikGapps-U/07-Nov-2023/

I saw others complaining about it being installed (YT original/vanced) so I used the config file to block its (possible) installation, personal preference as I prefer an... alternative.

It's also been cited as an example here ;
https://nikgapps.com/misc/2023/03/04/nikgapps.com/misc/2023/03/04/Uninstall-NikGapps.html

Rendering the assumption that it comes with a package.

**Comment by nikhilmenghani** [comment]
@jason-stream Full package doesn't contain YouTube or Music, Please follow the website and check with official support group on xda or telegram rather than believing random users.
![image](https://github.com/nikgapps/nikgapps.github.io/assets/3968443/02d709b2-29ca-4d80-b5be-be41e4be4364)

Also, the examples on the blogs are merely examples. They have no direct association with the definition of the packages.

Feel free to post any issue if you have questions and require clarity that website and faq doesn't already provides.

**Comment by jason-eu** [comment]
Fair enough, official XDA and Telegram it is.

Understood, I did not make the distinction between the examples on the blog and a real use case.

Thank you, I think that will be all, sorry if I was a bother.

I also just found the tutorial in the NikGApps telegram to make your own package and that sounds really fun.

Thank you for all your help and advice!

**Comment by jason-eu** [comment]
I found this, but it seems outdated and seems like an excellent place to get some descriptions ;

https://nikgapps.com/misc/2021/08/24/Packages-Definition.html

**Comment by nikhilmenghani** [comment]
> I found this, but it seems outdated and seems like an excellent place to get some descriptions ;
> 
> https://nikgapps.com/misc/2021/08/24/Packages-Definition.html

yes, this has been there since a long time - I thought it would be better to find the description right on the downloads page just by hovering on the package names. 

outdated? which packages are missing?

**Comment by jason-eu** [comment]
You're right, I agree it should be on the downloads page as well !

I mentioned it because I didn't know that it existed.

I noticed a few while modifying a config file, I have class now, but I'll post them when I get back to the library.

Okay, these are the ones missing from the FULL package ;
AndroidMigratePrebuilt
Flipendo
PixelLiveWallpapers
AdAway
YouTube
YouTubeMusic

Emoji Wallpaper isn't in this package description list, but that might due to the date of the page.

There might be more, but I hope I got all of them.

Each may have a specific reason as to why they're missing or present, but again this is kind of why I was asking about a different way to know what is what.

I am also not very technically inclined so please bear with me.

**Comment by nikhilmenghani** [comment]
> You're right, I agree it should be on the downloads page as well !
> 
> I mentioned it because I didn't know that it existed.
> 
> I noticed a few while modifying a config file, I have class now, but I'll post them when I get back to the library.
> 
> Okay, these are the ones missing from the FULL package ; AndroidMigratePrebuilt Flipendo PixelLiveWallpapers AdAway YouTube YouTubeMusic
> 
> Emoji Wallpaper isn't in this package description list, but that might due to the date of the page.
> 
> There might be more, but I hope I got all of them.
> 
> Each may have a specific reason as to why they're missing or present, but again this is kind of why I was asking about a different way to know what is what.
> 
> I am also not very technically inclined so please bear with me.

okay you probably don't have clarity on how NikGapps works, let me explain.

we have predefined packages (we call them variants) such as core, basic, omni, stock and full. Full doesn't include EACH app that we offer. In other words, the apps that are part of these packages are not the only apps we offer. The apps you listed above are available as addons.

addons - additional packages (individual ones) that you can flash on top of predefined variants. 

so for e.g. if you are satisfied with everything that basic offers and you are just missing one or two packages (say google maps, pixel launcher) you can flash basic zip + google maps and pixel launcher addons) - that way you will have everything you need.

I searched for the packages you listed and the definition is present for each one of them. If you don't see any description at https://nikgapps.com/misc/2021/08/24/Packages-Definition.html let me know.

**Comment by jason-eu** [comment]
You're right, I thought that the 'higher' the package variant was on the 'ladder', it meant that it contained the apps of the packages below it.

Of the add-ons ;
AndroidMigratePrebuilt
Flipendo
PixelLiveWallpapers
AdAway
YouTube
YouTubeMusic

I can only see YouTube and YouTube Music.


Thank you for helping me to understand and clarify my understanding, a little more, of how NikGApps functions.

I'm not anywhere near a master yet, even after flashing NikGapps as a priority since I had a little mido but I do hope, aside from confusion, and time permitting, there is a bit more clarity on what variant installs what.

Thank you again @nikhilmenghani

**Comment by nikhilmenghani** [comment]
@jason-eu you will find all the addons at https://sourceforge.net/projects/nikgapps/files/Releases/Addons-U/07-Nov-2023/
for android 14. Including 
AndroidMigratePrebuilt
Flipendo
PixelLiveWallpapers
AdAway

**Comment by jason-eu** [comment]
> @jason-eu you will find all the addons at https://sourceforge.net/projects/nikgapps/files/Releases/Addons-U/07-Nov-2023/ for android 14. Including AndroidMigratePrebuilt Flipendo PixelLiveWallpapers AdAway

They're not there, Nikhil, I checked before making the comment above.

![R1YKvHllRS](https://github.com/nikgapps/nikgapps.github.io/assets/13890252/6b5227c8-b2fd-4eb6-b829-4fa3de8f0e0e)
